content/post/mnist-example.md

```diff
@@ -0,0 +1,174 @@
++++
+title = "A Scikit-learn pipeline in Wallaroo"
+draft = false
+date = 2017-11-16T00:00:00Z
+tags = [
+    "wallaroo",
+    "machine learning",
+    "python",
+    "example",
+    "tutorial"
+]
+categories = [
+    "Machine Learning",
+    "Examples"
+]
+description = "Creating an inference pipeline with MNIST"
+author = "amosca"
++++
+
+Whilst it would seem that machine learning is taking over the world, a lot of the attention has been focused towards researching new methods and applications, and how to make a single model faster. At Wallaroo Labs we believe that, in order to make the benefits of machine learning truly ubiquitous, there needs to be a significant improvement in how we put those impressive models into production. This is where the stream computing paradigm becomes useful: as for any other type of computation, we can use streaming to apply machine learning models to a large quantity of incoming data, using available techniques in distributed computing.
+
+Nowadays, many applications with streaming data are either applying machine learning or have a very good use case for it. In this example, we will explore how we can build a machine learning inside Wallaroo, our high performance stream processing engine, to classify images from the [MNIST dataset](http://yann.lecun.com/exdb/mnist/), using a basic two-stage model in Python. Whilst recognizing hand-written digits is a practically solved problem, even a simple example like the one we are presenting provides a real use case (imagine automated cheque reading in a large bank), and the same setup can be used as a starting point for virtually any machine learning application - just replace the model.
+
+Everything has been implemented in the [current version of Wallaroo (1.4.0)](https://github.com/WallarooLabs/wallaroo/tree/0.4.0). The full code can be found on [GitHub](https://github.com/WallarooLabs/wallaroo_blog_examples/tree/master/sklearn-example). If you have any technical questions that this post didn't answer, or if you have any suggestions, please get in touch at [hello@WallarooLabs.com](mailto:hello@WallarooLabs.com), via [our mailing list](https://groups.io/g/wallaroo) or [our IRC channel](https://webchat.freenode.net/?channels=#wallaroo).
+
+We've been working on our processing engine, [Wallaroo](https://github.com/wallaroolabs/wallaroo/tree/release) for just under two years now. Our goal has been to make it as easy to build fast, scale-independent applications for processing data. When we open sourced Wallaroo last year we provided an API that let developers create applications using [Python](https://blog.wallaroolabs.com/2017/10/go-python-go-stream-processing-for-python/). The example discussed in this blog entry is written using that API.
```

Comment:
This paragraph doesn't seem to flow from the previous one. Not sure how to change things to make it flow better but wanted to mention it.

It would be better to point people to the blog post regarding the new Python API that doesn't use objects as that's what release `0.4.0` has.

Additionally, it might make sense to mention that Wallaroo also provides a `Go` API.